std_msgs/Header header
bool handReady
